lib/v2/screens/profile_screens/profile/interactor/viewmodels/profile_bloc.dart

```diff
@@ -15,6 +18,12 @@ class ProfileBloc extends Bloc<ProfileEvent, ProfileState> {
       var result = await GetProfileUseCase().run();
       yield ProfileStateMapper().mapResultToState(state, result);
     }
+    if (event is OnUpdateProfileImage) {
+      yield state.copyWith(pageState: PageState.loading);
+      var urlResult = await SaveImageUseCase().run(file: event.file);
+      var result = await UpdateProfileImageUseCase().run(imageUrl: urlResult.asValue.value, profile: state.profile);
+      yield UpdateProfileImageStateMapper().mapResultToState(state, result);
+    }
```

Comment:
@gguijarro-c-chwy Also all this part.

Reply:
yeeah, here we would check first if its a success or an error and react accordingly , but this if fine for now